cli/cmd/install.go

```diff
@@ -198,6 +198,11 @@ func installControlPlane(ctx context.Context, k8sAPI *k8s.KubernetesAPI, w io.Wr
 		return err
 	}
 
+	// the `--default-inbound-policy` flag configures the proxy override
+	// annotation, but in `install`, we also want to configure the default for
+	// the policy controller.
+	values.PolicyController.DefaultAllowPolicy = values.Proxy.DefaultInboundPolicy
```

Comment:
I'd recommend getting rid of the values.PolicyController.DefaultAllowPolicy value completely and instead use the values.Proxy.DefaultInboundPolicy value in the policy-controller template.  I don't think it makes sense for these two values to ever have different values.  Fewer values makes it easier for helm users too.